includes/models/class-accredible-learndash-model.php

```diff
@@ -34,7 +34,7 @@ public static function get_results( $where_sql = '', $limit = null, $offset = nu
 				$sql .= " OFFSET $offset";
 			}
 			// XXX `$where_sql` is a raw SQL so `$wpdb->prepare` cannot be used.
-			return $wpdb->get_results( $sql ); // phpcs:disable WordPress.DB.PreparedSQL.NotPrepared
+			return $wpdb->get_results( $sql ); // phpcs:ignore WordPress.DB.PreparedSQL.NotPrepared
```

Comment:
This change is not directly related to this task, but we need to use `phpcs:ignore` for inline comments unless adding `phpcs:enable` after `phpcs:disable`.